NEWS

```diff
@@ -1,3 +1,19 @@
+28-Jul-2017 IGNITION v0.17.2
+
+  Bug Fixes
+
+    - Fix failure to create files/directories/links on correct filesystem
+    - Fix issue where legacy force flag didn't force Ignition to decide to not
+      reuse existing filesystem
+    - Fix issue where Ignition would rely on the various mkfs utilities to not
+      overwrite existing filesystems, but some mkfs utilities would
+    - Use new enabled field when enabling systemd units in oem configurations to
+      avoid deprecation warning for the enable field
+    - Fix failure where hard link targets would be on incorrect filesystem,
+      causing creation to fail
+    - Fix failure to correctly compare filesystem UUIDs during check to reuse
+      existing filesystem, causing Ignition to not reuse the existing filesystem
```

Comment:
`Fix incorrect filesystem UUID check when deciding whether to reuse existing filesystem, causing Ignition to fail`?